README.md

```diff
@@ -1,16 +1,22 @@
+<!-- Code generated for API Clients. DO NOT EDIT. -->
+
 # ngrok API client library for Golang
-[![CI](https://github.com/ngrok/ngrok-api-go/actions/workflows/ci.yml/badge.svg)](https://github.com/ngrok/ngrok-api-go/actions/workflows/ci.yml)
```

Comment:
We don't want to delete this.

This shouldn't _really_ be a generated file, it only is since the example has imports (like `"github.com/ngrok/ngrok-api-go/v5"`) which vary based on the version.

I'll update the codegen to properly only update the example code.

Reply:
Fixing this as part of next internal changeset.